examples/Raytracer_Benchmark/Raytracer.h

```diff
@@ -0,0 +1,194 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+/// \file Raytracer.h
+/// \author Andrei Gheata (andrei.gheata@cern.ch)
+/// Adapted from VecGeom for AdePT by antonio.petre@spacescience.ro
+
+#ifndef VECGEOM_RAYTRACER_H_
```

Comment:
Name of the guard using RT_BENCHMARK instead of VECGEOM. The style should be consistent in all headers